.vscode/launch.json

```diff
@@ -11,9 +11,9 @@
 				"type": "mono"
 			},
 			"request": "launch",
-			"program": "${workspaceRoot}/OpenRA.Game.exe",
-			"cwd": "${workspaceRoot}",
-			"args": ["Game.Mod=cnc"],
+			"program": "${workspaceRoot}/bin/OpenRA.exe",
+			"cwd": "${workspaceRoot}/bin",
+			"args": ["Game.Mod=cnc", "Engine.EngineDir='..'"],
```

Comment:
Changing this to just `"Engine.EngineDir=.."` fixes the issue for me.

Reply:
You certainly need the `program` changes, too.